packages/material-ui/src/Link/Link.js

```diff
@@ -117,11 +117,6 @@ Link.propTypes = {
    * Either a string to use a DOM element or a component.
    */
   component: componentPropType,
-  /**
-   * The target of the link. You can read about this property on the
-   * [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Attributes)
-   */
-  target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top']),
```

Comment:
Fix W3C warning: https://validator.w3.org/nu/?doc=https%3A%2F%2Fmaterial-ui.netlify.com%2Fstyle%2Flinks%2F

![capture d ecran 2019-01-12 a 13 14 45](https://user-images.githubusercontent.com/3165635/51073079-21bfae80-166c-11e9-9dbb-6942d919814c.png)

Reply:
What I mean by this is that we don't validate props that we simply forward.

`target` is valid DOM attribute for `a` elements just not `button`. So the validation was correct for the default case but not if `component="button"` was used. The validator mentioned this explicitly "Attribute `target` not allowed on element `button` at this point.".

Oh and we need to remove the default value for `target`: https://github.com/mui-org/material-ui/blob/dfff366df7b83b26b2ab6cc169e51184742d043b/packages/material-ui/src/Link/Link.js#L143